mayastor/tests/error_count.rs

```diff
@@ -35,12 +36,19 @@ static ERROR_DEVICE: &str = "error_device";
 static EE_ERROR_DEVICE: &str = "EE_error_device"; // The prefix is added by the vbdev_error module
 static BDEV_EE_ERROR_DEVICE: &str = "bdev:///EE_error_device";
 
+static YAML_CONFIG_FILE: &str = "/tmp/error_count_test_nexus.yaml";
+
 #[test]
 fn nexus_error_count_test() {
     common::truncate_file(DISKNAME1, 64 * 1024);
     common::truncate_file(DISKNAME2, 64 * 1024);
 
-    test_init!();
+    let mut config = Config::default();
+    config.err_store_opts.enable_err_store = true;
+    config.err_store_opts.err_store_size = 256;
+    config.write(YAML_CONFIG_FILE).unwrap();
+
+    test_init!(Some(YAML_CONFIG_FILE.to_string()));
```

Comment:
Its not some to see Some(macro!()) perhaps, give the testing, you want to unwrap in the macro?

Reply:
Moved the string handling into the macro.